components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -0,0 +1,215 @@
+                               attribute:NSLayoutAttributeWidth
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:nil
+                               attribute:NSLayoutAttributeNotAnAttribute
+                              multiplier:1
+                                constant:width].active = YES;
+
+  if (!_inkTouchController) {
+    _inkTouchController = [[MDCInkTouchController alloc] initWithView:self];
+    [_inkTouchController addInkView];
+  }
+
+  _imageView = [[UIImageView alloc] init];
+  [self.contentView addSubview:_imageView];
+  _imageView.translatesAutoresizingMaskIntoConstraints = NO;
+  _imageView.alpha = kImageAlpha;
+  [NSLayoutConstraint constraintWithItem:_imageView
+                               attribute:NSLayoutAttributeTop
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:self.contentView
+                               attribute:NSLayoutAttributeTop
+                              multiplier:1
+                                constant:16.f].active = YES;
+  [NSLayoutConstraint constraintWithItem:_imageView
+                               attribute:NSLayoutAttributeLeading
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:self.contentView
+                               attribute:NSLayoutAttributeLeading
+                              multiplier:1
+                                constant:16.f].active = YES;
+  [NSLayoutConstraint constraintWithItem:_imageView
+                               attribute:NSLayoutAttributeWidth
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:nil
+                               attribute:NSLayoutAttributeNotAnAttribute
+                              multiplier:1
+                                constant:24.f].active = YES;
+  [NSLayoutConstraint constraintWithItem:_imageView
+                               attribute:NSLayoutAttributeHeight
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:nil
+                               attribute:NSLayoutAttributeNotAnAttribute
+                              multiplier:1
+                                constant:24.f].active = YES;
+  _widthConstraint.constant = CGRectGetWidth(self.contentView.frame) - (kLeadingPadding * 2);
+}
+
+- (void)layoutSubviews {
+  [super layoutSubviews];
+
+  NSMutableParagraphStyle *paragraphStyle = [[NSMutableParagraphStyle alloc] init];
+  paragraphStyle.hyphenationFactor = 1.f;
+  NSDictionary<NSString *, id> *attributes = @{ NSParagraphStyleAttributeName : paragraphStyle };
+  NSMutableAttributedString *attributedString =
+  [[NSMutableAttributedString alloc] initWithString:_itemAction.title attributes:attributes];
+  _textLabel.attributedText = attributedString;
+
+  CGFloat leadingConstant;
+  if (_itemAction.image == nil) {
```

Comment:
Rather than testing against nil, it is common practice to check for existence via:
if (_itemAction.image)  {
  // Exists
} else {
  // Doesn't exist
}

Reply:
Done